gitCoreJGit/src/main/java/com/virtuslab/gitcore/impl/jgit/GitCoreBranch.java

```diff
@@ -24,22 +22,22 @@
 import com.virtuslab.gitcore.api.BaseGitCoreBranch;
 import com.virtuslab.gitcore.api.BaseGitCoreCommit;
 import com.virtuslab.gitcore.api.GitCoreException;
-import com.virtuslab.gitcore.api.GitCoreNoSuchBranchException;
 import com.virtuslab.gitcore.api.GitCoreNoSuchCommitException;
+import com.virtuslab.gitcore.api.GitCoreNoSuchRevisionException;
 import com.virtuslab.logger.IPrefixedLambdaLogger;
 import com.virtuslab.logger.PrefixedLambdaLoggerFactory;
 
 @Getter
 @RequiredArgsConstructor
-@ToString
```

Comment:
Whoops... so basically no `toString` is provided now?...

Reply:
yup, we do not use it anyway